audio-worklet/test/bitcrusher-test.js

```diff
@@ -101,23 +98,26 @@ context.startRendering()
       let unalteredOutput = buffer.getChannelData(1);
 
       // Allow for fractional error beyond audible perception. This error
-      // occurs because any sample passing through bitcrusher will undergo 
+      // occurs because any sample passing through bitcrusher will undergo
       // manipulation in Math.round() and will therefore be represented by a
-      // new floating point number that differs slightly from the original. 
-      // In tested  samples, the maximum observed error is on the order of 10^-7 
-      // and most frequently between 0 and 10^-8. The permitted error is set
-      // here to be 10^-6 so that tests pass.
-      const permittedSampleError = Math.pow(10,-6);
-
-      // First script-processed-buffer is all zero.
-      const latency = bufferLength;
+      // new floating point number that differs slightly from the original.
+      // In tested  samples, the maximum observed error is 3.46451997756958e-7
+      const permittedSampleError = 3.46451997756958e-7 + 1e-7;
```

Comment:
Note for the next push that this number has changed ever so slightly to accommodate a different (44.1k) sampling rate.